### PR TITLE
Add ECMA-402 proposals

### PR DESCRIPTION
**Pull request by chicoxyzzy**
According to https://github.com/tc39/proposals/issues/76

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
@ljharb do you mean that ECMA-262 proposals should remain in root and ECMA-402 should be in subdirectory? Where should be a link to ECMA-402 proposals in that case? Somewhere after [ECMAScript proposals list](https://github.com/tc39/proposals#ecmascript-proposals)?

IMHO having proposals to both specs on one page is very handy.

**Comment by chicoxyzzy** [comment]
https://github.com/chicoxyzzy/proposals/tree/add_ecma402_proposals

**Comment by ljharb** [comment]
@chicoxyzzy yes; that's what i mean - that they'd be on two separate lists. I'd expect a link to the 402 proposals to appear prominently at the top of the main readme, though.

**Comment by chicoxyzzy** [comment]
PR updated

File: ecma420/finished-proposals.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Finished Proposals
+
+Finished proposals are proposals that have reached stage 4, and are included in the [latest draft](https://tc39.github.io/ecma402/) of the specification.
+
+| Proposal                                                                 | Champion                           | TC39 meeting notes | Expected Publication Year |
+|--------------------------------------------------------------------------|------------------------------------|--------------------|---------------------------|
+| [Intl.DateTimeFormat.prototype.formatToParts][]                          | Zibi Braniecki                     | [July 2016][Intl.DateTimeFormat.prototype.formatToParts-notes] | 2017
+| [Intl.NumberFormat.prototype.formatToParts][]                            | Zibi Braniecki                     | [September 2017][Intl.NumberFormat.prototype.formatToParts-notes] | 2018
+| [Intl.PluralRules][]                                                     | Caridy Patiño, Eric Ferraiuolo     | [September 2017][Intl.PluralRules]        | 2018
```
Comment (chicoxyzzy) [inline ecma420/finished-proposals.md]: please review Expected Publication Year column
cc @ljharb @caridy

**Comment by chicoxyzzy** [comment]
420 😅

File: ecma402/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+### Onboarding existing proposals
+
+Proposals that are Stage 1 and above must be transferred to [the TC39 GitHub organization](https://github.com/tc39) for discoverability and archival purposes. To onboard a proposal that lives outside the TC39 organization:
+
+1. Transfer your repository to the [@tc39-transfer](http://github.com/tc39-transfer) organization
+  - if you are a TC39 delegate, but not an admin in that organization, please contact @LJHarb
+2. @bterlson will transfer your repository to the TC39 organization the next chance he gets.
+
+Note that as part of the onboarding process your repository name may be normalized. Don't worry, redirects will continue to work.
```
Comment (littledan) [inline ecma402/README.md]: This isn't really true; GitHub pages just breaks. However, this is the process anyway.

File: ecma402/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Please see [Contributing to ECMAScript](/CONTRIBUTING.md) for the most up-to-date information on contributing proposals to this standard.
+
+### Onboarding existing proposals
+
+Proposals that are Stage 1 and above must be transferred to [the TC39 GitHub organization](https://github.com/tc39) for discoverability and archival purposes. To onboard a proposal that lives outside the TC39 organization:
+
+1. Transfer your repository to the [@tc39-transfer](http://github.com/tc39-transfer) organization
+  - if you are a TC39 delegate, but not an admin in that organization, please contact @LJHarb
+2. @bterlson will transfer your repository to the TC39 organization the next chance he gets.
```
Comment (littledan) [inline ecma402/README.md]: I can do this too if needed.

File: ecma402/README.md (hunk 2 above)
Comment (ljharb) [reply ecma402/README.md]: Yeah true; this (which was copypasted from the main readme) should be clarified to say that repo URLs will redirect and github pages will break.

**Comment by ljharb** [comment]
Thanks! Let's merge this as-is; we can make followup PRs to explore combining the 402 lists into one, and to clarify the transfer process in both 262 and 402.

**Pull request merged**